commonknowledge/mapped
# Add Posthog for TCC account

## Description
This PR adds a Posthog instance that is linked to a separate Climate Coalition managed account. 
There is a domain check so it only sends events to the TCC account that come from the domain peopleclimatenature.org

## Motivation and Context
Addresses issue [CC-165](https://linear.app/commonknowledge/issue/CC-165/create-a-posthog-account-for-tcc)

## How Can It Be Tested?
Download the branch and run locally
Set up your Next.js .env file by copying .env.example - message me for API keys
Set the NEXT_PUBLIC_ENVIRONMENT to production for testing purposes
Click around on https://peopleclimatenature.org/ and observe events are being sent to TCC Climate Coalition Posthog account (log in details in Bitwarden)
Click around on https://prototype.mapped.commonknowledge.coop/ and observe events are being sent to the main Mapped Posthog account (log in details in Bitwarden)

## How Will This Be Deployed?
Addition of env variables:
NEXT_PUBLIC_POSTHOG_KEY_MAIN=
NEXT_PUBLIC_POSTHOG_KEY_SECONDARY=

## Types of changes
